docs/DeveloperGuide.adoc

```diff
@@ -295,59 +317,144 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |student with many content-heavy modules|easily search for any flashcards that are relevant to my modules|more efficient with my time
+
+|`* * *`|student taking many modules|easily search for a deck that I want to access|more efficient with my time
+
+|`* * *` |student |delete decks of the modules that I am no longer taking|be more organised
+
+|`* * *` |student |edit my flashcards |change or add extra information whenever I want to
+
+|`* *`|student who receives flashcard images from her friends|drag and drop the images into the application|conveniently create new flashcards
+
+|`* *`|student who owns multiple mobile/computer devices|access flashcards on all my devices|revise them on the go
+
+|`* *`|student with short attention span|play memory games in the application|remember my key concepts better
 
-|`* * *` |user |add a new person |
+|`* *`|student who likes designing and aesthetics|customise the colours and fonts of the flashcards|
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* *` |student |set priority levels for the flashcards I am less familiar with|it will appear more often during future test sessions
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* *`|student taking timed examinations|time myself during a test session|better prepared to think under timed conditions
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *`|student|revisit previous test sessions|continue my revision
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* *`|student|tag flashcards with different topics|filter and revise the topics that I am less familiar with
+
+|`*`|student who constantly travels|view my created cards online|revise them on my mobile devices
+
+|`*`|student with a short attention span|add animations to my flashcards|remain entertained
+
+|`*`|student taking modules with other friends|collaborate on decks with other users|help each other revise the content
+
+|`*`|student|edit other people's decks|fill in any gaps in my knowledge
+
+|`*`|student who likes to store content to study on her phone|convert the flashcards to images|so that I can refer to them easily
+
+|`*`|student who needs incentive|earn rewards|will be motivated to use the flashcards more
+
+|`*`|competitive student|graded on a bell-curve with other Computer Science students who are using the same application|see how well I have revised compared to the rest
+
+|`*`|student who likes to study with her friends|send my friends flashcards that I created|share my flashcards with them
+
+|`*`|student who is lazy to type out long questions|read out the questions and answers to the flashcards to automatically add them via voice recognition|
```

Comment:
This user story might not be very specific to the target audience as the target audience was previously defined as users who can type fast and prefers typing.